include/votca/xtp/md2qmengine.h

```diff
@@ -18,7 +18,7 @@
 #ifndef VOTCA_XTP_MD2QMENGINE_H
 #define VOTCA_XTP_MD2QMENGINE_H
 
-#include <votca/csg/topology.h>
+#include <votca/csg/csgtopology.h>
```

Comment:
why the renaming they are in separate namespaces

Reply:
Will revert.